testplan/parser.py

```diff
@@ -409,6 +418,14 @@ def generate_parser(self) -> HelpParser:
             ),
         )
 
+        report_group.add_argument(
```

Comment:
move this arg right below/above `dump-failed-tests` instead? maybe we can change the place of `pdf-style` as well?